docs/WIP/Tutorial_Motif_only_Matrix_Profile.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.                    dist = amp[i] - ip[i] - ip[j]
</pre><p>Since <code>amp</code> contains z-normalized distances here, IMO it makes sense to say that <code>ip[i]</code> and <code>ip[j]</code> should be z-normalized distances. Therefore, I think the distance computed in <code>computeKTIP</code> should be the z-normalized distance. Better to check paper to make sure of it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/1046/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>